tests/gis_tests/test_postgis_extension.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+
+from django.db import connection
+from django.test import TestCase
+
+
+@unittest.skipUnless(connection.vendor == 'postgresql', "PostgreSQL specific tests")
+class TestsPostGISCreateExtension(TestCase):
+
+    def setUp(self):
+
+        """make sure the extension is not available initially"""
+
+        supported = [
```

Comment:
Not sure how this could or should be tested - if it even is needed for testing.. its kinda wonky right now

Reply:
You can assume that PostGIS is enabled, otherwise, the tests won't run at all.